_cities/milan.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Milan
+location: "Milan, Italy"
+tagline: "Pizza, code and fine wine!"
+mailing_list: team.milan
```

Comment:
I couldn't find such a list in our domain.  Could you drop an email to team.tech@opentechschool.org so we don't forget about setting it up?